### ISSUE TITLE
Circular economy vision shouldn't be a required field

### ISSUE THREAD

**Issue by fannyBkane** [emfoundation/ce100-app#1484]
![screen shot 2018-08-23 at 11 47 10](https://user-images.githubusercontent.com/20577339/44521676-c6e0e700-a6cb-11e8-980d-ec92bcdf5830.png)

**Comment by asimonw** [comment]
@fannyBkane I think we had initially decided that we'd want this to be required, but this is an interesting use case.. This will only happen if you, as an admin, want to update the logo after the org was already created, but before a primary user has gone in to add a mission statement. I.e. the first time an admin creates the org with logo, this is not an issue.

I'll add it to the backlog to be reviewed.

**Comment by fannyBkane** [comment]
> I think we had initially decided that we'd want this to be required

Do you remember why?

**Comment by asimonw** [comment]
Because we thought that an org profile with empty mission statement would look "empty" and would be less useful. So this was just to encourage members to provide a mission statement. It did not take into account the situation you found yourself in. So, good to review.

**Comment by asimonw** [comment]
Yes, that could be an improvement 👍

**Comment by Kimberley-Heath** [comment]
I agree, sounds good! It doesn't need to be a required field unless it is part of the initial set up of their account, but as it stands, if they go to edit they have the intention of filling it out anyway. Would be good to add some guidelines too as you'll see that all members who entered their vision on their own accord, wrote a sentence rather than a paragraph.
